docs/reference/src/documentation/operations/reentrancy.md

```diff
@@ -14,19 +14,19 @@ Sway provides a stateless [re-entrancy](https://fuellabs.github.io/sway-libs/boo
 To use the guard we must import it.
 
 ```sway
-{{#include ../../code/operations/re_entrency/src/main.sw:import}}
+{{#include ../../code/operations/re_entrancy/src/main.sw:import}}
```

Comment:
Updating this breaks the path and will not display properly

Reply:
we should probably create another issue for this; could be a good first issue too

Reply:
@zees-dev can you share where is this page live, I can look into it.

Reply:
Actually i think we can just change the filename here too in the same PR - along with the typo fix; thoughts @bitzoic ?